src/components/Tabset/index.js

```diff
@@ -206,38 +189,39 @@ export default class Tabset extends Component {
                 <ul
                     className={this.getInnerContainerClassName()}
                     role="tablist"
-                    style={this.getTabsetStyles()}
                     onKeyDown={this.handleKeyPressed}
+                    onScroll={this.updateTabsetButtonsVisibility}
                     ref={this.tabsetRef}>
 
                     <Provider value={{
                         activeTabName,
                         onSelect,
-                        privateRegisterTab: this.registerTab,
-                        privateUnRegisterTab: this.unRegisterTab,
+                        privateRegisterTab,
+                        privateUnRegisterTab,
                         fullWidth,
                     }}>
```

Comment:
I meant the entire object:
```
const context = {
                        activeTabName,
                        onSelect,
                        privateRegisterTab: this.registerTab,
                        privateUnRegisterTab: this.unRegisterTab,
                        privateRegisterTab,
                        privateUnRegisterTab,
                        fullWidth,
                    }
```
then:
```
<Provider value={context}>
```
remember not js inside jsx
separation of concerns